lisdf/components/model_mjcf.py

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env python3
+# -*- coding: utf-8 -*-
+# File   : model_mjcf.py
+# Author : Jiayuan Mao
+# Email  : maojiayuan@gmail.com
+# Date   : 03/24/2022
+#
+# This file is part of lisdf.
+# Distributed under terms of the MIT license.
```

Comment:
We should discuss on Slack whether we should include this or not. Since the author, email and date be determined by git blame and the license is in the root of the repo.

Reply:
Yes. In general I suggest putting a few lines at the top... Could be our license, or project name, or something... Even just a single line of #! /usr/bin/env python3...

When I open the editor if my python file starts from the first line... I will feel very very uncomfortable... That's probably my only insist..... LOLLL